README.md

```diff
@@ -56,3 +41,94 @@ How to create superuser
 ```shell
 python3 manage.py createsuperuser
 ```
+
+
+# Changes to database schema
```

Comment:
In README.md Changes to database schema , it seem forgot to mention the change of iperson_id , iwork_id become ForeignKey of CofkCollect

Reply:
I discuss this under ## Collect table changes, perhaps it is not clear enough?